packages/core/src/commands/remove-node.ts

```diff
@@ -0,0 +1,63 @@
+import type { NodeType, ResolvedPos } from '@prosekit/pm/model'
+import type { Command } from '@prosekit/pm/state'
+
+import type { CommandCreator } from '../types/command'
+import { getNodeType } from '../utils/get-node-type'
+
+/**
+ * Returns a command that removes the given type of node at the current selection or at
+ * the given position.
+ *
+ * @public
+ */
+export function removeNode(options: {
+  /**
+   * The type of the mark to remove.
+   */
+  type: string | NodeType
+
+  /**
+   * The position of the document. By default it will be the anchor position of current selection.
+   */
+  position?: number
```

Comment:
Let's use `pos` instead of `position`. `pos` matches both prosekit [^1] and prosemirror's [^2] naming conventions.

[^1]: https://github.com/prosekit/prosekit/blob/b136cd73b22a7a8eb5cb5fc3a924d2f02407b180/packages/core/src/commands/set-node-attrs.ts#L24-L28
[^2]: https://prosemirror.net/docs/ref/#view.EditorView.posAtCoords

Reply:
Renamed to `pos`.